docs/src/examples/Cloudy_example.md

```diff
@@ -0,0 +1,82 @@
+# Cloudy Example
+
+### What Is This Example About?
+
+This example is based on Cloudy, a microphysics model that simulates how cloud droplets collide and coalesce into larger drops. Collision-coalescence is a crucial process for the formation of rain. 
+
+Cloudy takes two inputs: The initial mass distribution of the cloud droplets, and a so-called collision-coalescence kernel, which specifies the probability that two droplets of masses x and y collide and coalesce in a given time interval. 
+Cloudy simulates the evolution of the initial cloud droplet mass distribution over time, as more and more droplets combine into bigger drops. It does this not by simulating each individual droplet, but by using a "bulk approach", which only keeps track of some number of moments of the cloud droplet mass distribution.
+
+This example shows how Ensemble Kalman methods can be used to learn the parameter of the initial cloud droplet mass distribution from observations of the moments of that mass distribution at a later time. The collision-coalescence kernel is assumed to be known, but one could also learn the parameters of the kernel instead of the parameters of the droplet distribution (or both).
+
+Cloudy is used here in a "perfect model" (aka "known truth") setting, which means that the "observations" are generated by Cloudy itself, by running it with the true parameter values. In more realistic applications, the observations will come from some external measurement system.
+
+For more information on Cloudy, see [here](https://github.com/CliMA/Cloudy.jl.git).
+
+
+### Prerequisites
+
+In order to run this example, you need to install `Cloudy.jl` (the "#master"
+lets you install the current master branch):
+```
+pkg > add Cloudy#master
+```
+
+### Structure
+
+The file `DynamicalModel.jl` provides the functionality to run Cloudy, i.e., to map the parameters of the initial cloud droplet mass distribution to the (zeroth-, first-, and second-order) moments of the distribution at a specified end time. In this example, there is no post-processing required to produce the observations from the model output (i.e., the observation map ``\mathcal{H}`` mentioned in the [docs](https://clima.github.io/EnsembleKalmanProcesses.jl/dev/ensemble_kalman_inversion/) is the identity map).
+
+The file `Cloudy_example_eki.jl` sets up the inverse problem and solves it using [Ensemble Kalman inversion](https://clima.github.io/EnsembleKalmanProcesses.jl/dev/ensemble_kalman_inversion/), and the file `Cloudy_example_uki.jl` does the same using unscented Kalman inversion.
```

Comment:
Can you either capitalize Ensemble and Unscented or neither?

Reply:
Thanks for catching, I went for "ensemble".